openapi/paths/aml-checks.yaml

```
@@ -0,0 +1,31 @@
+get:
+  x-products:
+    - Core
+  tags:
+    - AML
+  summary: Retrieve a collection of Customer AML checks
+  operationId: GetAmlCheckCollection
+  x-sdk-operation-name: getAll
+  description: |-
+    Retrieve a collection of recorded AML checks against Customers.
+    These checks are records of Customer data and potentially matching data in our AML list.
```

issue: [start all operations with an active verb](https://github.com/Rebilly/api-definitions/blob/main/WRITING-STYLE.md#operations)

```suggestion
  description: |-
    Retrieves a list of customer AML checks.
    These checks are records of customer data and potentially matching data in AML lists.
```